src/spawn.rs

```diff
@@ -0,0 +1,78 @@
+//! Entity spawning utilities.
+
+// Unused utilities may trigger these lints undesirably.
+#![allow(dead_code)]
+
+use bevy::ecs::system::EntityCommand;
+use bevy::ecs::system::EntityCommands;
+use bevy::prelude::*;
+
+/// Re-exported extension traits.
+#[allow(unused_imports)]
+pub mod prelude {
+    pub use super::{AddExt as _, SpawnExt as _, WorldSpawnExt as _};
+}
+
+/// An extension trait that provides helper functions for deferred entity spawning.
+pub trait SpawnExt {
+    // Workaround for https://github.com/bevyengine/bevy/issues/14231#issuecomment-2216321086.
+    /// Spawn an entity with an [`EntityCommand`].
+    fn spawn_with<M: 'static>(&mut self, command: impl EntityCommand<M>) -> EntityCommands;
+}
+
+impl SpawnExt for Commands<'_, '_> {
+    fn spawn_with<M: 'static>(&mut self, command: impl EntityCommand<M>) -> EntityCommands {
+        let mut e = self.spawn_empty();
+        e.add(command);
+        e
+    }
+}
+
+impl SpawnExt for ChildBuilder<'_> {
+    fn spawn_with<M: 'static>(&mut self, command: impl EntityCommand<M>) -> EntityCommands {
+        let mut e = self.spawn_empty();
+        e.add(command);
+        e
+    }
+}
+
+/// An extension trait that provides helper functions for immediate entity spawning.
+pub trait WorldSpawnExt {
+    // Workaround for https://github.com/bevyengine/bevy/issues/14231#issuecomment-2216321086.
+    /// Spawn an entity with an [`EntityCommand`].
+    fn spawn_with<M: 'static>(&mut self, command: impl EntityCommand<M>) -> EntityWorldMut;
+}
+
+impl WorldSpawnExt for World {
+    fn spawn_with<M: 'static>(&mut self, command: impl EntityCommand<M>) -> EntityWorldMut {
+        let mut e = self.spawn_empty();
+        e.add(command);
+        e
+    }
+}
+
+impl WorldSpawnExt for WorldChildBuilder<'_> {
+    fn spawn_with<M: 'static>(&mut self, command: impl EntityCommand<M>) -> EntityWorldMut {
+        let mut e = self.spawn_empty();
+        e.add(command);
+        e
+    }
+}
+
+/// An extension trait that supports arbitrary [`EntityCommand`]s after `world.spawn()`.
+pub trait AddExt {
+    // Workaround for https://github.com/bevyengine/bevy/issues/14278.
```

Comment:
```suggestion
    // Workaround for <https://github.com/bevyengine/bevy/issues/14278>.
```
Link is currently broken because of the period.

Reply:
Where does the link not work? It works for me in VS Code like this.

Reply:
Clicking it on my Firefox redirects me to this page: https://github.com/bevyengine/bevy/issues/14278.?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen
I think your VS Code is just smart enough to figure out that the dot is not actually part of the URL, while my browser / GitHub assumes it is.